esignet-service/pom.xml

```diff
@@ -13,7 +13,7 @@
 
 	<groupId>io.mosip.esignet</groupId>
 	<artifactId>esignet-service</artifactId>
-	<version>1.1.0-SNAPSHOT</version>
```

Comment:
Seems like your develop branch is not upto date, pull the latest updates and then raise the PR